examples/transforms/transform.rs

```diff
@@ -127,7 +127,7 @@ fn rotate_cube(
     // Update the rotation of the cube(s).
     for (mut transform, cube) in &mut cubes {
         // Calculate the rotation of the cube if it would be looking at the sphere in the center.
-        let look_at_sphere = transform.looking_at(center, transform.local_y());
+        let look_at_sphere = transform.looking_at(center, *transform.local_y());
```

Comment:
We could technically make `looking_at` take an `impl Into<Vec3>` to remove the need for deref here, but it's probably follow-up material (and also pretty niche probably)